R/arcadia.R

```diff
@@ -23,9 +23,7 @@
 #'   theme_arcadia()
 #' }
 
-theme_arcadia <- function(x_axis_type = "numerical", y_axis_type = "numerical", background = TRUE) {
-  # load all Suisse fonts and store
-  loaded_suisse_fonts <- load_arcadia_fonts()
+theme_arcadia <- function(x_axis_type = "numerical", y_axis_type = "numerical", background = TRUE, padding = c(.25, .25, .25, .25)) {
```

Comment:
It appears as though padding isnt' documented as a param, but I may be missing something.

Reply:
addressed